app/exposureNotificationsNativeModule.ts

```diff
@@ -48,13 +48,13 @@ export const simulatePositiveDiagnosis = async (
   debugModule.simulatePositiveDiagnosis(cb);
 };
 
-export const disableExposureNotifications = async (
+export const toggleExposureNotifications = async (
```

Comment:
is it necessary to need to go to the native layer for this? could we have some state on the ExposureNotificationsContext of a boolean? in fact i believe we already have this.

Reply:
I think this will become toggle instead of disable anyways going forward, so not sure if it is a big deal if we keep for now.

Reply:
probably nbd!